drivers/usb/device/Kconfig

```diff
@@ -142,6 +142,22 @@ config USB_MCUX
 	help
 	  NXP MCUX USB Device Controller Driver for MXRT and LPC SoC's.
 
+config USB_MCUX_ENDPOINT_BUFFER_SIZE
+	int "Size of intermediate buffer in USB driver"
+	default 512
+	range 64 512
```

Comment:
Is 512 the maximum MCUX can use? The USB 2.0 allows up to 1024 on high-speed interrupt endpoint.

Reply:
okay, I changed this